docs/content/tutorials/kic/egress-walkthrough.md

```diff
@@ -47,7 +47,7 @@ If you want to view metrics for NGINX Ingress Controller, ensure that you have d
       nginx-meshctl deploy ... --mtls-mode=strict --disabled-namespaces=legacy
```

Comment:
We're gonna have to change this guide up (and anywhere else that may reference `disabled-namespaces`). This example should deploy the mesh with `--disable-auto-inject` and then set `--enabled-namespaces` with the necessary values (I think just the `default` namespace).